wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -3271,3 +3282,6 @@ instance HasDiagnostic WalletError where
             "msg"
         RequestThrottled{} ->
             "microsecondsUntilRetry"
+
+instance Arbitrary NoContent where
+    arbitrary = pure NoContent
```

Comment:
Meh, still one orphan exchanged for another :disappointed:  ... 
Would it be too much work to go for a newtype here, or at leat pin-point the place where it's actually needed ?

Reply:
```
[7 of 9] SwaggerSpec

/home/pawel/IOHK/cardano-sl/wallet-new/test/SwaggerSpec.hs:57:13: error:
    • No instance for (Arbitrary NoContent)
        arising from a use of ‘validateEveryToJSON'’
    • In the second argument of ‘($)’, namely
        ‘validateEveryToJSON' (Proxy @V1.API)’
      In the second argument of ‘($)’, namely
        ‘describe "(V1) ToJSON matches ToSchema"
           $ validateEveryToJSON' (Proxy @V1.API)’
      In a stmt of a 'do' block:
        parallel
          $ describe "(V1) ToJSON matches ToSchema"
              $ validateEveryToJSON' (Proxy @V1.API)
   |
57 |             validateEveryToJSON' (Proxy @ V1.API)
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Arbitrary instances should live with their type definitions IMO. 

Otherwise, orphan arbitrary instances are one of the few justifiable ones, even to the Anti Orphan Squad

Reply:
Yup. It seems to be actually the "accepted pattern" here (in IOHK) to have orphan Arbitrary instances defined in test code only (even though, I tend to prefer having everything about a type defined next to the type itself).

In that specific case, since we need this Arbitrary only in the SwaggerSpec, let's make it an orphan instance there and not in `V1.Types`.